src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/PolicyCmdletBase.cs

```diff
@@ -49,6 +74,23 @@ protected PSObject[] GetOutputObjects(string resourceType, params JToken[] resou
                 });
         }
 
+        /// <summary>
+        /// Converts the resource object to specified resource type object.
+        /// </summary>
+        /// <param name="resources">The policy definition resource object.</param>
+        protected PSObject[] GetFilteredOutputObjects(string resourceType, ListFilter filter, params JToken[] resources)
+        {
+            return resources
+                .CoalesceEnumerable()
+                .Where(resource => resource != null)
+                .SelectArray(resource =>
+                {
+                    var psobject = resource.ToResource().ToPsObject();
+                    psobject.Properties.Add(new PSNoteProperty(resourceType, psobject.Properties["ResourceId"].Value));
+                    return psobject;
+                }).Where(result => (filter == ListFilter.None || string.Equals(((PSObject)result.Properties["Properties"].Value).Properties["PolicyType"].Value.ToString(), filter.ToString(), StringComparison.OrdinalIgnoreCase))).ToArray();
```

Comment:
Be null safe on PolicyType here in case Azure Stack is running an old version that doesn't have policyType (don't think that is possible but who knows).

Reply:
Done